arch/arm64/src/common/arm64_mmu.c

```diff
@@ -426,7 +426,7 @@ static void init_xlat_tables(const struct arm_mmu_region *region)
   uint64_t level_size;
 
 #ifdef CONFIG_MMU_DEBUG
-  sinfo("mmap: virt %llx phys %llx size %llx\n", virt, phys, size);
+  sinfo("mmap: virt %lu phys %lu size %lu\n", virt, phys, size);
```

Comment:
Why did we change this? I believe `virt` and `phys` are Memory Addresses, so it might be better to print them as hex?

Reply:
good catch, i fixed the datatype but removed the `x` just added it back.